Experimental/src/org/sleuthkit/autopsy/experimental/volatilityDSP/VolatilityProcessor.java

```diff
@@ -177,23 +193,20 @@ private void runVolatilityPlugin(String pluginToRun) throws VolatilityProcessorE
         commandLine.add("\"" + executableFile + "\""); //NON-NLS
         File memoryImage = new File(memoryImagePath);
         commandLine.add("--filename=" + memoryImage.getName()); //NON-NLS
-
-        File imageInfoOutputFile = new File(moduleOutputPath + "\\imageinfo.txt"); //NON-NLS
-        if (imageInfoOutputFile.exists()) {
-            String memoryProfile = parseImageInfoOutput(imageInfoOutputFile);
-            commandLine.add("--profile=" + memoryProfile); //NON-NLS
+        if (profile.isEmpty() == false) {
```

Comment:
This is flagged as an unnecessary comparison by Codacy, and I agree. It can be just like line 142 above.